### PR TITLE
Add Complementary Grass Fixes

### PR DESCRIPTION
**Pull request by Ghos71y**
Edit due to request: The version number for the checksum is 1.0

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -7778,6 +7778,16 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         nav: 10
 
+  - name: 'Complementary Grass Fixes.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/71562'
```
Comment (sibir-ine) [inline masterlist.yaml:7783]: Links should have trailing slashes if possible:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/71562/'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
For cleaning data, please provide the version number for the checksum in the commit message or PR description so future maintainers & contributors can know what it's from. Looks good otherwise.

**Comment by Ghos71y** [comment]
It took me a minute to figure it out but the suggestion should be added in the most recent commit.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**